vllm_ascend/quantization/w8a8_dynamic.py

```diff
@@ -779,7 +866,9 @@ def apply(
                 moe_all_to_all_group_name=self.moe_all_to_all_group_name,
                 log2phy=log2phy,
                 global_redundant_expert_num=global_redundant_expert_num,
-                shared_experts=shared_experts)
+                shared_experts=shared_experts,
+                quantized_x_for_share=shared_gate_up,
```

Comment:
How about directly using `shared_gate_up` as the field name?

`quantized_x_for_share` doesn't quite fit the semantics in this context.